programs/bpf_loader/src/syscalls.rs

```diff
@@ -82,6 +85,8 @@ pub enum SyscallError {
     TooManyAccounts,
     #[error("Overlapping copy")]
     CopyOverlapping,
+    #[error("return data too large ({0} > {1})")]
```

Comment:
caps, and would you please fix the other two lower case instances while you are in here?

Reply:
done